scripts/sanity_chk/sanitycheck-platform-schema.yaml

```diff
@@ -18,6 +18,9 @@ mapping:
   "type":
     type: str
     enum: [ 'mcu', 'qemu', 'sim', 'unit', 'native']
+  "emulation":
```

Comment:
Does 'simulation' make more sense instead of 'emulation'?

Reply:
Doesn't QEMU stand for "Quick Emulator"? 

either way, I do not really care how we call it, the distinction here is needed for real platforms such as the mps2-an385 that can run under an emulator, the binary basically can be flashed to a real device as opposed to some qemu configurations we have which are configured to be run under qemu only.